### PR TITLE
Insights

### PR DESCRIPTION
**Pull request by thecodemonkey**
+ some basic insights
+ scheduled Job posts the insights every night on the slack channel

### CODE REVIEW

File: src/common/services/import/lokalwirkt/mapping.ts [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+            }
+        }        
+    }
+
+    return out;
+}
+
+
+
+// "community_centre", -
```
Comment (christianaschoff) [inline src/common/services/import/lokalwirkt/mapping.ts:74]: srsly?

Comment (thecodemonkey) [reply src/common/services/import/lokalwirkt/mapping.ts:74]: i know it's a wrong PR, sorry ✌️

**Comment by thecodemonkey** [comment]
just a question, can someone approve? I would like to integrate this pr into the development branch to continue there.

**Comment by buecheko** [comment]
> just a question, can someone approve? I would like to integrate this pr into the development branch to continue there.

I will have a look. I don't fully understand it yet. So this might take a while.

**Pull request merged**